pkg/util/http.go

```diff
@@ -164,9 +164,17 @@ func ParseProtoReader(ctx context.Context, reader io.Reader, expectedSize, maxSi
 		err = proto.NewBuffer(body).Unmarshal(req)
 	}
 	if err != nil {
+		if sp != nil {
+			sp.LogFields(otlog.String("event", "util.ParseProtoRequest[unmarshal done]"), otlog.Error(err))
```

Comment:
nit: Apparently `otlog` has an `Event()` method that does exactly this.

Reply:
And now it's used in latest version of the PR. Thanks!